rezerwacje-app/src/main/java/org/niewidoczniakademicy/rezerwacje/core/csv/RoomMapper.java

```diff
@@ -0,0 +1,24 @@
+package org.niewidoczniakademicy.rezerwacje.core.csv;
+
+import org.niewidoczniakademicy.rezerwacje.core.model.database.Room;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class RoomMapper {
+    public Room convert(org.niewidoczniakademicy.rezerwacje.core.model.csv.Room room) {
```

Comment:
Why is the "full" path for Room there? Is is intended?

Reply:
As above, I convert CSV -> DB objects.